app/routes/categories.$.tsx

```diff
@@ -54,7 +54,9 @@ export default function Tags() {
                   <div className="padding-bottom-24">
                     {selectedTag.questions.length === 0
                       ? 'No pages found'
-                      : `${selectedTag.questions.length} pages tagged "${selectedTag.name}"`}
+                      : selectedTag.questions.length === 1
+                        ? `1 page tagged "${selectedTag.name}"`
+                        : `${selectedTag.questions.length} pages tagged "${selectedTag.name}"`}
```

Comment:
this kind of nested tenary operations is cumbersome. Could you extract it out of the `return` into a proper `if` or something?